app/src/main/java/com/example/playlistmaker/MediaActivity.kt

```diff
@@ -44,27 +59,43 @@ class MediaActivity : AppCompatActivity() {
         roundedCorners = (dpToPx(widthDisplay.toFloat(), this)) / cornersRatio
```

Comment:
🍏 Можно использовать стандартные инструменты. R.dimen.album_cover_corner_radius - сохраненное в ресурсах значение радиуса в dp:
```suggestion
        roundedCorners = resources.getDimensionPixelSize(R.dimen.album_cover_corner_radius)
```